src/main/java/Duke.java

```diff
@@ -1,10 +1,115 @@
+import java.util.Scanner;
+
 public class Duke {
     public static void main(String[] args) {
+        Scanner sc = new Scanner(System.in);
+        DukeList list = new DukeList();
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        String bound = "____________________________________________________________";
+        String topBound = "____________________________________________________________\n";
+        String bottomBound = "\n____________________________________________________________\n";
+
+        System.out.println(topBound + logo + "\nHello! I'm Duke\n" + "What can I do for you?\n" + bottomBound);
+
+        while (sc.hasNextLine()) {
+            String input = sc.nextLine();
+            String[] arr = input.split("/");
```

Comment:
String array name can be more descriptive :D

Reply:
Agree with you, since the scope of arr is quite big.